Exercise_5.java

```diff
@@ -2,19 +2,38 @@
     void swap(int arr[], int i, int j) 
     { 
 	//Try swapping without extra variable 
+    int temp = arr[i]; 
+    arr[i] = arr[j]; 
+    arr[j] = temp; 
     } 
   
     /* This function is same in both iterative and 

```

Comment:
Good Job! Please add time and space complexities about your approach. Review inline comments.